### PR TITLE
[Doc] Add Link Prediciton to Use Your Own Graph tutorial

### PR DESCRIPTION
**Pull request by zhjwy9343**
*Issue #, if available:*

*Description of changes:*
This PR include changes for adding Link Prediction examples into the Use Your Own Graph tutorial

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/source/tutorials/own-data.rst [hunk 1]
```diff
@@ -73,29 +73,16 @@ Once succeeded, the command will create a set of folders and files under the ``/
 
 The input configuration JSON
 ```````````````````````````````
-GraphStorm's graph construction tool relies on the configuration JSON to provide graph information. Explanations of the format of the configuration JSON contents could be found in the :ref:`GraphStorm Graph Configuration JSON <gconstruction-json>`. Below show the contents of the examplary ACM `config.json` file.
+The above command automatically creates the examplary ACM `config.json` file, some of which are listed below.
 
-.. code-block:: json
+.. code-block:: yaml
```
Comment (classicsong) [inline docs/source/tutorials/own-data.rst:78]: Isn't it a json file?

File: docs/source/tutorials/own-data.rst [hunk 2]
```diff
@@ -276,30 +193,34 @@ The raw node and edge data files are both in a parquet format, whose contents ar
 .. figure:: ../../../tutorial/ACM_raw_parquet.png
     :align: center
 
-In this example, only the ``paper`` nodes have labels and the task is node classification. So, in the JSON file, the ``paper`` node has the ``labels`` field, and the ``task_type`` is specified as ``classification``. Correspondingly, in the paper node parquet file, there is a column, ``label``, stores the label values. All edge types do not have features associated. Therefore, we only have two columns in these parquet files for edges, the ``source_id`` and the ``dest_id``.
+In this example, only the ``paper`` nodes have labels and the task is node classification. So, in the JSON file, the ``paper`` node has the ``labels`` field, and the ``task_type`` is specified as ``classification``. Correspondingly, in the paper node parquet file, there is a column, ``label``, stores the label values. All edge types do not have features associated. Therefore, we only have two columns in these parquet files for edges, the ``source_id`` and the ``dest_id``. For the link prediction task, there is no actual labels. Users just need to specify the ``labels`` field in one or more ``edge`` objects of the JSON.
```
Comment (classicsong) [inline docs/source/tutorials/own-data.rst:196]:
```suggestion
In this example, only the ``paper`` nodes have labels and the task is node classification. So, in the JSON file, the ``paper`` node has the ``labels`` field, and the ``task_type`` is specified as ``classification``. Correspondingly, in the paper node parquet file, there is a column, ``label``, stores the label values. All edge types do not have features associated. Therefore, we only have two columns in these parquet files for edges, the ``source_id`` and the ``dest_id``. For the link prediction task, there is no actual labels. Users just need to specify the ``labels`` field in one or more ``edge`` objects of the JSON config file.
```

File: docs/source/tutorials/own-data.rst [hunk 3]
```diff
@@ -309,8 +230,8 @@ Because the above command specifies the ``--num-parts`` to be ``1``, there is on
 
 .. note::
 
-    - Because the parquet format has some limitations, such as only supporting 2 billion elements in a column, etc, we suggest users to use HDF5 format for very large dataset.
-    - The two mapping files, ``node_mapping.pt`` and ``edge_mapping.pt``, are used to record the mapping between the ogriginal node and edge ids in the raw data files and the ids of nodes and edges in the constructed graph. They are important for mapping the training and inference outputs. Therefore, DO NOT move or delete them.
+    - Because the parquet format has some limitations, such as only supporting 2 billion elements in a column, etc, we suggest users to use HDF5 format for very large datasets.
+    - The two mapping files, ``node_mapping.pt`` and ``edge_mapping.pt``, are used to record the mapping between the ogriginal node and edge ids in the raw data files and the ids of nodes and edges in the GraphStorm node ID space. They are important for mapping the training and inference outputs back to the original input data. Therefore, **DO NOT** move or delete them.
```
Comment (classicsong) [inline docs/source/tutorials/own-data.rst:234]: Graph node ID space. Let align the naming with https://github.com/awslabs/graphstorm/pull/621

File: docs/source/tutorials/own-data.rst [hunk 4]
```diff
@@ -12,30 +12,30 @@ It is easy for users to prepare their own graph data and leverage GraphStorm's b
...
 
 Step 1: Prepare Your Own Graph Data
 -------------------------------------
 There are two options to prepare your own graph data for using GraphStorm:
 
-- Prepare your graph in the required raw data format, and use GraphStorm's construction tools to automatically generate the input files. **This is the preferred method**.
-- Prepare your data as a DGL heterogeneous graph following the required format, and then use GraphStorm's partition tools to generate the input files. This option is for experienced DGL users.
+- Prepare your graph in the required raw data format, and use GraphStorm's construction tools to automatically generate the input files. **This is the preferred method**, as GraphStorm provides :ref:`distributed data processing and construciton tools<gs-processing>` to handle extreme large graph data.
+- Prepare your data as a DGL heterogeneous graph following the required format, and then use GraphStorm's partition tools to generate the input files. This option is for experienced DGL users and relatively small graph data.
```
[the remark this answers is not in the document]
Comment (zhjwy9343) [reply docs/source/tutorials/own-data.rst:22]: The required format will be described in the following sections.

So, add addition words to make things clear.

File: docs/source/tutorials/own-data.rst (hunk 1 above)
Comment (zhjwy9343) [reply docs/source/tutorials/own-data.rst:78]: Yes, it is a json. But in rst files, the rst interpreter can render yaml better for JSON contents. So I change it to yaml.

File: docs/source/tutorials/own-data.rst (hunk 2 above)
Comment (zhjwy9343) [reply docs/source/tutorials/own-data.rst:196]: changed.

File: docs/source/tutorials/own-data.rst (hunk 3 above)
Comment (zhjwy9343) [reply docs/source/tutorials/own-data.rst:234]: Sure, changed.

**Review by classicsong — APPROVED** [review]

**Pull request merged**